src/main/java/io/camunda/testing/assertions/ProcessInstanceAssert.java

```diff
@@ -461,4 +516,17 @@ public ProcessInstanceAssert hasVariableWithValue(final String name, final Strin
         .map(Record::getValue)
         .collect(Collectors.toMap(VariableRecordValue::getName, VariableRecordValue::getValue));
   }
+
+  protected static class ProcessInstanceActual {
```

Comment:
:question: What is the purpose of this class?

Reply:
I was overthinking this yesterday. I decided to introduce this class so we could have an entry point to `ProcessInstanceAssert` that wouldn't be a `ProcessInstanceEvent`. Therefore I created this class, but since we only use the process instance key we might as well use the key itself as the actual.